### DISCUSSION TITLE
[DatePickerInput] type="range" - Allow range of 1 day

### DISCUSSION THREAD

**Discussion by gamegee** [mantinedev/mantine#6130] [Feature requests]
Hi, thanks for the amazing lib :)
 
I noticed the [DatePickerInput](https://mantine.dev/dates/date-picker-input/#dates-range) component in `type="range"` mode, doesn't allow to select a range of 1 day.

If we take [Airbnb as an example](https://www.airbnb.fr/?_set_bev_on_new_domain=1710775600_OGEzMDFlZTdhZGU2), picking a range of days is usual, but choosing just one is fine too. This flexibility suits different needs, like spontaneous stays or quick stopovers.

**Comment by rtivital** [comment]
https://mantine.dev/dates/date-picker/#single-date-in-range

**Reply by gamegee** [reply]
Thanks, I just found it this morning by taking a closer eye on the docs 😄
